curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md

```diff
@@ -0,0 +1,419 @@
+---
+id: 6690e10ebe2181212abc9652
+title: Design a Business Card
+challengeType: 14
+dashedName: design-a-business-card
+demoType: onClick
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. **Do not copy this demo project**
```

Comment:
```suggestion
**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
```
As I understand we are not going to add that in the description of each project.

Reply:
I'm going to commit this so we can get this merged. I don't think any of the other projects have it. We can go through and make sure they are consistent later as well.